port/win/port_win.h

```diff
@@ -128,50 +128,25 @@ class CondVar;
 
 class Mutex {
  public:
-
-   /* implicit */ Mutex(bool adaptive = kDefaultToAdaptiveMutex)
-#ifndef NDEBUG
-     : locked_(false)
-#endif
-   { }
-
+  /* implicit */ Mutex(bool adaptive = false);
```

Comment:
The thing that this is not clear to me is the reason why we need to change any files in port/win directory. This is a directory that rocksdb-cloud inherits from facebook/rocksdb, and my assumption is that it is best that rocksdb-cloud do not need to modify most of the code that it inherits from rocksdb.

Reply:
Probably best indeed that you stick to the original RocksDB code.

Reply:
are you saying that the code in port/win/port_win.h has diverged from upstream rocksdb's port_win.h?

Reply:
No, I'm saying I have a patch of port/win/port_win.h that was never submitted back to RocksDB, and that it's irrelevant to this PR, so I should remove it from the PR.